### PR TITLE
Filter UI Added

### PR DESCRIPTION
**Pull request by richardshaju**

### CODE REVIEW

**Comment by richardshaju** [comment]
Don't need to worry about this conflict. It occurred because I formatted the code.

File: app/page.tsx [hunk 1]
```diff
@@ -215,17 +261,55 @@ export default function Home() {
...
+          className={` my-7 mt-7  mb-28 justify-center  flex md:flex-row  ${data.length == 0 ? "opacity-0" : ""}`}
+        >
           <p className="text-3xl">Upcoming Events</p>
+          <div className="md:left-28 z-30 absolute md:my-7 my-12  text-sm md:text-lg p-2">
+            <div className="flex flex-row items-center gap-1 md:gap-2">
+              <LuFilter size={25} /> Filter
+              <details className="dropdown">
+                <summary className="m-1 btn bg-transparent border-2" ref={clubDropdownButton} >{clubDropdown}</summary>
+                <ul  className={`p-2 shadow menu dropdown-content z-[1] bg-base-100 rounded-box w-52`}>
+                  {getClubs.map((club, idx) => (
```
Comment (chethaslp) [inline app/page.tsx:294]: Error: Missing "key" prop for element in iterator.

File: app/page.tsx [hunk 2]
```diff
@@ -215,17 +261,55 @@ export default function Home() {
...
+                  ))}
+                </ul>
+              </details>
+        
+   
+              <details className="dropdown ">
+                <summary className="m-1 btn bg-transparent border-2" ref={typeDropdownButton}>{typeDropdown}</summary>
+                <ul className={`p-2 shadow menu dropdown-content z-[1] bg-base-100 rounded-box w-52 `}>
+                  {['Online', 'Offline', 'Both'].map((type, idx)=>(
+                    <li>
```
Comment (chethaslp) [inline app/page.tsx]: Error: Missing "key" prop for element in iterator.

File: app/page.tsx [hunk 3]
```diff
@@ -45,89 +65,98 @@ export default function Home() {
...
+            );
           }
         }
-      })
-      }
-      reqNotification();
-  }),[])
+      });
+    }
+    reqNotification();
```
Comment (chethaslp) [inline app/page.tsx]: Check latest commit and update this code to resolve PR conflict.

**Review by chethaslp — CHANGES_REQUESTED** [review]

**Comment by chethaslp** [comment]
Deployment Failing.

```
./app/page.tsx
Error: 
  x Expected ';', '}' or <eof>
     ,-[/vercel/path0/app/page.tsx:95:1]
  95 |         return () => clearInterval(intervalId); // Cleanup function to clear the interval when component unmounts or when bannerEvent changes
  96 |       }, [bannerEvent]);
  97 |     
  98 | ,->   useEffect(() => {
  99 | |       function reqNotification() {

.........................................................................

138 | |               );
 139 | |             }
 140 | |           }
 141 | |         })
 142 | |         }
 143 | |         if(window.Notification) reqNotification();
 144 | |->   }),[])
     : `---       ^
     : `---- This is the expression part of an expression statement
 145 |     
 146 |     
 147 |       useEffect(() => {
     `----
```

**Pull request merged**